src/Views/Search/Search.js

```diff
@@ -1,11 +1,66 @@
-import React from 'react'
-
-const Search  = () => {
-    return ( 
-        <div>
-            Search
-        </div>
-      );
-}
- 
-export default Search;
\ No newline at end of file
+import { useState } from "react";
+import Book from "./Book";
+
+const Search = () => {
+  const [searchTitle, setSearchTitle] = useState("");
+  const [searchLocation, setSearchLocation] = useState("");
+  const [books, setBooks] = useState([]);
+  const [status, setStatus] = useState("Let's start searching!");
+
+  const handleInputChange = (e) => {
+    if (e.target.id === "searchTitle") return setSearchTitle(e.target.value);
+    if (e.target.id === "searchLocation")
+      return setSearchLocation(e.target.value);
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setStatus("Searching...");
+    searchBooks();
+  };
+
+  const searchBooks = async () => {
+    let res = await fetch(
+      "https://pikabook.herokuapp.com/api/books?" +
+        new URLSearchParams({
+          name: searchTitle,
+          location: searchLocation,
+        })
+    );
+    if (!res.ok) {
+      res = await res.text();
+      status(res);
+    } else {
+      res = await res.json();
+      setBooks(res);
+      if (books.length === 0) setStatus("No books found");
+    }
```

Comment:
I think that logic associated with fetching data shouldn't lay in React components. Better place for such code would be some external file (i.e. called `searchBookService.jsx`) to increase readability. What do you think about it? 

Also did you mean `setStatus(res)` instead of `status(res)`? :)